bouncer/bouncer/settings.py

```diff
@@ -0,0 +1,120 @@
+"""
+Django settings for bouncer project.
+
+Generated by 'django-admin startproject' using Django 2.0.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '4k!2%^fq=ia$nv+e^s*heuo7ej6-xf!k60ojernxafq!w6=l&b'
```

Comment:
It would be better to create a directory named `.secrets`, put this directory in `.gitignore` and load the value of `SECRET_KEY` from a file named `.secrets/secret-key`.

This way, we will not expose the actual key we're using to Git, which is an anti-pattern.